### PR TITLE
Extract all GH issue for ingestion

### PR DESCRIPTION
**Pull request by pankajastro**
closes: https://github.com/astronomer/ask-astro/issues/113

### CODE REVIEW

**Review by Lee-W — APPROVED** [review]

**Review by sunank200 — CHANGES_REQUESTED** [review]
We should do the following before we merge this change:
1. Run a fresh bulk ingestion DAG and run this DAG as well in Dev database.
2. Request @vatsrahul1001 to test this to make sure the performance has not degraded. This should be deployed in dev slackbot.
Blocking the merge till then

**Comment by pankajastro** [comment]
@vatsrahul1001 can you please test this once from your side?

**Review by sunank200 — APPROVED** [review]

**Pull request merged**
